part2/send_data.py

```diff
@@ -0,0 +1,92 @@
+import pandas as pd
```

Comment:
What do you think about using a description of the specific task for each instead of `part1` and `part2` for the folder names?

We could, for instance, have everything related to the project itself in a `project` folder and then the two parts would be something like `project/patient_representation` and `project/phenotyping`. 

Happy to hear other suggestions as well.

Reply:
Using description sounds good. :+1: